lls/main.c

```diff
@@ -209,6 +210,58 @@ submit_nd(struct rte_mbuf **pkts, unsigned int num_pkts,
 	return 0;
 }
 
+/*
+ * Match the packet if it fails to be classifed by ACL rules,
+ * if it's an ND packet, then submit it to the LLS block.
+ *
+ * Return values: -ENOENT for no matching, -1 for submit_nd error,
+ * and 0 for successful match and submit.
+ */
+static int
+match_nd(struct rte_mbuf *pkt, struct gatekeeper_if *iface)
+{
+	/*
+	 * The ND header offset in terms of the
+	 * beginning of the IPv6 header.
+	 */
+	int nd_offset;
+	uint8_t nexthdr;
+	struct ether_hdr *eth_hdr =
+		rte_pktmbuf_mtod(pkt, struct ether_hdr *);
+
+	if (likely(rte_be_to_cpu_16(eth_hdr->ether_type) == ETHER_TYPE_IPv6)) {
```

Comment:
1. Turn this a simple equality defining constant `BE_ETHER_TYPE_IPv6`.

2. Invert test and avoid indentation.